x/evm/types/params.go

```diff
@@ -54,6 +60,7 @@ func DefaultParams() Params {
 		EvmDenom:     ethermint.AttoPhoton,
 		EnableCreate: true,
 		EnableCall:   true,
+		ExtraEIPs:    []int(nil), // TODO: define default values
```

Comment:
enable EIP1344 will make uniswapV2 runnable, EIP1344 was add opcode support of `CHAINID` to make users getting chain-id in contract

Reply:
this PR is only for adding support for extra eips. The default values will be discussed separately on #633

Reply:
i think, to fix #626 and #632 , use #645 is better way than #643